content/docs/command-reference/gc.md

```diff
@@ -120,6 +121,19 @@ project we want to clear.
   them (each project is a path) to keep data that is currently referenced from
   them.
 
+- `--not-in-remote` - keep cached objects that are _not_ in the remote. This
+  will remove the objects from the local cache that have been pushed and are
+  present in the remote.
+
+  For objects using the
+  [`remote` field](/doc/user-guide/project-structure/dvc-files#output-entries),
+  the check will be against that remote.
+
+  For objects not using the
+  [`remote` field](/doc/user-guide/project-structure/dvc-files#output-entries),
+  the check will be against the default remote unless a specific one is given
+  with `-r`.
```

Comment:
Does the same apply for `-c`?

Reply:
Updated with the same description. not sure if it should be phrased differently for the `cloud` case or not